docs/en/status-communities/import-a-status-community.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 445
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Import a Status Community
+
+To import a Status Community, you use the community's public key. This key is a large string of characters that helps you find the community you want to join. To make sharing and importing communities easier, Status includes the community's public key in a QR code or link, like this one: `https://join.status.im/c/zQ3shMtRomXyM5svciMpdK7zDaR12XDeQWbzi4Fb8E6YywDrE`.
```

Comment:
I'd add a sentence introducing in what cases users will want to import a community, for example, "If someone shares a Status Community with you, you can import the community to quickly find it".
Because if without context, I'd assume this is what a community owner can do.

Reply:
Thanks, @cheny0 -- I include this information in the paragraph right after that one ;)

> Importing a community can be helpful when you can't find the community or someone shares the community QR code or link.